ParabolInc/parabol
# feat: Add eventType filter in timeline view

## Description
There are two renames that contribute to the bigger PR:
1. `packages/client/utils/useUserTaskFilters.ts` → `packages/client/utils/useQueryParameterParser.ts`
2. `packages/client/utils/constructTeamFilterQueryParamURL.ts` → `packages/client/utils/constructFilterQueryParamURL.ts`


## Demo
![2023-06-30 11 13 23](https://github.com/ParabolInc/parabol/assets/1879975/688426d3-617e-49ca-a9dd-53830850b8e5)

## Testing scenarios

- [ ] Filter certain event type
  - Go to timeline page
  - See the filter menu should default to "All Events"
  - Select an event type from the menu and see corresponding events displayed

- [ ] Back to "All Events"
  - When an event type is selected & filtered from the menu, open the menu again to see "All Events" option present

- [ ] Other team filters
  - Verify team & user filters in Team, Tasks & Meetings Page still work as expected

## Final checklist
